src/lib/Messages.js

```diff
@@ -255,67 +255,67 @@ class Messages {
     }
   };
 
-  tryfromBinary = <TType>(buffer: Buffer, typeName: string): ?TType => {
-      var result = null;
+  tryFromBinary = (buffer: Buffer, typeName: string): ?mixed => {
+      let result = null;
       try {
         result = this.fromBinary(buffer, typeName);
-      } catch (exception) {
-        logger.error('Could not parse type: ${typeName} ${buffer}', exception);
+      } catch (error) {
+        logger.error('Could not parse type: ${typeName} ${buffer}', error);
       }
       return result;
   };
 
-  fromBinary = <TType>(buffer: Buffer, typeName: string): TType => {
```

Comment:
Why change this?  Making this polymorphic should work... I'd prefer not to have to have mixed :/